code/obj/machinery/manufacturer.dm

```diff
@@ -294,43 +293,28 @@
 			dat += "<small>"
 			if (src.category != "Downloaded")
 				for(var/datum/manufacture/A in src.available)
-					var/list/mats_used = material_check(A)
+					var/list/mats_used = get_materials_needed(A)
```

Comment:
You might want to type this and the other mats_used exist since you can?

Reply:
`get_materials_needed(A)` returns an associative list of `mat_id` values, so I can't.